lib/components/STable.vue

```diff
@@ -9,6 +9,8 @@ import {
   toRefs,
   watch
 } from 'vue'
+import { DynamicScroller, DynamicScrollerItem } from 'vue-virtual-scroller'
+import 'vue-virtual-scroller/dist/vue-virtual-scroller.css'
```

Comment:
I earlier added it to bootstrap.css but it's not being imported in VitePress so styles were breaking there 👀

Reply:
I'll add this to VitePress 👍 In case if it doesn't work, I'll just place it here 😅